package.json

```diff
@@ -47,6 +47,7 @@
     "level": "1.7.0",
     "npm-check-updates": "^2.12.1",
     "nsp": "^2.8.0",
+    "oembed-parser": "^1.0.8",
```

Comment:
we need security review for this one. I see that there is no new usage, where we just missing it from the dependencies?

cc @diracdeltas

Reply:
https://github.com/brave/internal/issues/203

Reply:
we should not use ombed-parser in general since it uses Node's networking stack (https://github.com/ndaidong/oembed-parser/blob/master/src/utils/fetchEmbed.js), but this usage is ok for now since it only uses the provider.json rules file